### PR TITLE
Oyster fixes

### PR DESCRIPTION
**Pull request by vishwajeetraj11**
- [x] Clear button same size as input next to it in marketplace 
- [x] Select input colors fixes in marketplace filters
- [x] modal scroll fix

### CODE REVIEW

File: src/lib/atoms/v2/componentClasses.ts [hunk 1]
```diff
@@ -38,5 +38,5 @@ export const inputClasses = {
 	inputText:
 		'input input-ghost h-[30px] w-full text-body p-0 font-semibold text-primary disabled:opacity-70 disabled:text-primary disabled:placeholder:text-primary/[.2] focus-within:text-primary placeholder:text-primary/[.2] focus:outline-none focus-within:border-b-2 focus:bg-transparent',
 	searchInputText:
-		'input input-ghost h-[30px] w-full text-[15px] p-0 font-normal text-primary focus:outline-none focus-within:border-b-2 focus-within:text-primary focus:bg-transparent disabled:text-blue-500 placeholder-[#A8A8A8]'
+		'input input-ghost h-[30px] w-full text-[16px] p-0 font-normal text-[#030115] focus:outline-none focus-within:border-b-2 focus-within:text-[#030115] focus:bg-transparent disabled:text-blue-500 placeholder-[#A8A8A8]'
```
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/atoms/v2/componentClasses.ts:41]: done

File: src/lib/components/v2/inputs/AmountInputWithTitle.svelte [hunk 2]
```diff
@@ -34,14 +35,15 @@
...
+		if (showLabelFocused) return;
 		onBlurAction();
 	};
 
 	const handleChange = (e: any) => {
 		handleUpdatedAmount(e);
 		if (e && e.target && e.target.id && e.target.id === 'pond-input-amount-Duration') {
 			const elementToFocus: HTMLElement | null = document.getElementById(`title-cost`);
-
+			// if (showLabelFocused) return;
```
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/components/v2/inputs/AmountInputWithTitle.svelte:46]: done

File: src/lib/page-components/v2/bridge/modals/SuccessfulConversionModal.svelte [hunk 3]
```diff
@@ -26,7 +26,7 @@
 		conversionFrom === 'pond' ? pondToMPond(amountConverted) : mPondToPond(amountConverted);
 </script>
 
-<Modal {modalFor}>
+<Modal {modalFor} padding={false}>
```
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/bridge/modals/SuccessfulConversionModal.svelte:29]: done

File: src/lib/page-components/v2/oyster/inventory/modals/JobDetailsModal.svelte [hunk 4]
```diff
@@ -131,7 +131,7 @@
 		</div>
 	</svelte:fragment>
 	<svelte:fragment slot="actionButtons">
-		<div class="w-full">
+		<div class="w-full pt-6">
```
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/oyster/inventory/modals/JobDetailsModal.svelte:134]: we have it in the modal component. its only needed at some places explicitely.

[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/oyster/inventory/modals/JobDetailsModal.svelte:134]: not all modals need it.

File: src/lib/page-components/v2/oyster/marketplace/OysterMarketplaceFilters.svelte [hunk 5]
```diff
@@ -79,7 +79,7 @@
 		<Button
 			variant="filled"
 			size="medium"
-			styleClass="w-[140px] font-normal font-poppins"
+			styleClass="w-[140px] h-auto font-normal font-poppins"
```
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/oyster/marketplace/OysterMarketplaceFilters.svelte:82]: that's something we don't want everywhere.

File: src/lib/page-components/v2/oyster/inventory/modals/JobDetailsModal.svelte (hunk 4 above)
[the remark this answers is not in the document]
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/oyster/inventory/modals/JobDetailsModal.svelte]: fixed it

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**